test/e2e/specs/editor/various/draggable-blocks.spec.js

```diff
@@ -0,0 +1,151 @@
+			// Move the dragged handle a little bit (to the top left of the handle's
+			// original location) to trigger the dragging chip to appear.
+			position: { x: 0, y: 0 },
+			// Bypass the "actionability" checks because the handle will disappear.
+			force: true,
+		} );
+
+		await expect(
+			page.locator( 'data-testid=block-draggable-chip >> visible=true' )
+		).toBeVisible();
+
+		// Move to and hover on the upper half of the paragraph block to trigger the indicator.
+		const firstParagraph = page.locator(
+			'role=document[name="Paragraph block"i] >> text=1'
+		);
+		const firstParagraphBound = await firstParagraph.boundingBox();
+		await page.mouse.move( firstParagraphBound.x, firstParagraphBound.y );
+		await firstParagraph.hover( {
+			position: { x: 0, y: 0 },
+		} );
+
+		await expect(
+			page.locator( 'data-testid=block-list-insertion-point-indicator' )
+		).toBeVisible();
+
+		// Drop the paragraph block.
+		await page.mouse.up();
+
+		await expect.poll( editor.getEditedPostContent )
+			.toBe( `<!-- wp:paragraph -->
+<p>2</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>1</p>
+<!-- /wp:paragraph -->` );
+	} );
+
+	test( 'can drag and drop to the bottom of a block list', async ( {
+		editor,
+		page,
+	} ) => {
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '1' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '2' );
+		await page.keyboard.press( 'ArrowUp' );
+
+		// Confirm correct setup.
+		await expect.poll( editor.getEditedPostContent )
+			.toBe( `<!-- wp:paragraph -->
+<p>1</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>2</p>
+<!-- /wp:paragraph -->` );
+
+		await editor.showBlockToolbar();
+
+		const dragHandle = page.locator(
+			'role=toolbar[name="Block tools"i] >> role=button[name="Drag"i][include-hidden]'
+		);
+		// Hover to the center of the drag handle.
+		await dragHandle.hover();
+		// Start dragging.
+		await page.mouse.down();
+		await dragHandle.hover( {
+			// Move the dragged handle a little bit (to the top left of the handle's
+			// original location) to trigger the dragging chip to appear.
+			position: { x: 0, y: 0 },
+			// Bypass the "actionability" checks because the handle will disappear.
+			force: true,
+		} );
+
+		await expect(
+			page.locator( 'data-testid=block-draggable-chip >> visible=true' )
+		).toBeVisible();
+
+		// Move to and hover on the bottom half of the paragraph block to trigger the indicator.
+		const secondParagraph = page.locator(
+			'role=document[name="Paragraph block"i] >> text=2'
+		);
+		const secondParagraphBound = await secondParagraph.boundingBox();
+		await page.mouse.move(
+			secondParagraphBound.x,
+			secondParagraphBound.y + secondParagraphBound.height * 0.75
+		);
+		await secondParagraph.hover( {
+			position: {
+				x: 0,
+				y: secondParagraphBound.height * 0.75,
+			},
+		} );
+
+		await expect(
+			page.locator( 'data-testid=block-list-insertion-point-indicator' )
+		).toBeVisible();
```

Comment:
It'd be good to assert the position of the drop indicator, as this is something that has regressed in the past.

Reply:
100%. I pushed the commented-out code to show that it was passing without this assertion.

Reply:
The puppeteer tests check that the y coordinate of the indicator is correct, so something like that would be good (if we can get them passing):
https://github.com/WordPress/gutenberg/blob/fd3b08bef5da71ca02db872ddd57a7a77beaec21/packages/e2e-tests/specs/editor/various/draggable-block.test.js#L74-L76

Reply:
Added the position check.